common/src/main/webpack/dev.webpack.config.js

```diff
@@ -47,7 +47,8 @@ const Web = Merge(
         rewrites: [
           { from: /^\/conf.json$/, to: '/conf/development.conf.json' },
         ]
-      }
+      },
+      https: true
```

Comment:
I guess you get a cert warning on the browser but that's ok, we could consider making an explore CA to issue cents for local development

Reply:
It bothers the first time only. Unfortunately, there's not way to configure WDS to serve both http and https. I guess we could issue a cert for `local.lucuma.xyz`, yes.